docs/index.md

```
@@ -11,7 +11,349 @@ Look for the "Section A: Guide" section in the {% else %} branch.
+:class-header: sd-text-center sd-fs-5 sd-align-minor-center sd-font-weight-bold sd-text-capitalize
+:class-body: sd-text-center sd-fs-5
+:class-footer: text-smaller
+The CrateDB Guide
+^^^
+{material-outlined}`hiking;3.5em`
++++
+Guides and tutorials about how to use CrateDB in practice.
+:::
+
+:::{grid-item-card}
+:link: https://learn.cratedb.com/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)